doc/intro.md

```diff
@@ -84,6 +81,23 @@ Three alternatives are available to produce population projections:
 
 This method involves projecting births, deaths and migrations by age and sex.
 
+See below the minimal cohort component method we aim to provide:
+
+![minimal CCM](images/MVP_generic_CCM_flowchart.png)
+
+Note:
+
+We will ultimately provide four alternatives for projecting the components of change (i.e. fertiliy, mortality and migration): using fixed values / rates or using trend values / rates.
+See below:
+
+![CCM - projection alternatives](images/Component_projection_alternatives.png)
+
+For estimating the value or rate for the first year of projection (jump-off year or base year), we will provide three alternatives. See below:
+
+![CCM - base year alternatives](images/Jump_off_year_calculation_alternatives.png)
+
+
+
 1) Fertility
 
 The standard methodology for birth projections relies on applying age-specific fertility rates (ASFR)
```

Comment:
I find the bullet points below confusing, because they are listing data sources but the opening sentence here is about methodology. Remove for now, or make it clearer why these data sources are being listed?